Template/Tag/CustomHtmlTag.web.js

```diff
@@ -155,23 +161,24 @@
             var limit = 5000; // prevent endless loop
             var counter = 0;
             var child;
+            var previousElement = '';
 
             while (counter in children && children[counter] && counter < limit) {
                 child = children[counter];
                 counter++;
 
                 if (isJavaScriptElement(child)) {
                     // we have to re-create the element, otherwise wouldn't be executed
-                    insertNode(parent, cloneScript(child), append);
+                    previousElement = insertNode(parent, cloneScript(child), append);
```

Comment:
none of the attributes are yet passing the previousElement to `insertNode`?

Reply:
I actually pushed to show how the changes would look and was experimenting without `previousElement` and that got pushed by mistakenly, added back